### DISCUSSION TITLE
大世界深渊海域完成后总是不能正常退出地图

### DISCUSSION THREAD

**Discussion by jonexhd** [LmeSzinc/AzurLaneAutoScript#3590] [General]
RT，每次深渊海域打完之后直接就卡死，从日志判断是点击次数太多自动重启了。。请问下有什么办法回避吗？调120帧也没用

**Comment by LmeSzinc** [comment]
如何上报bug How to Report Bugs
在提问题之前至少花费 5 分钟来思考和准备，才会有人花费他的 5 分钟来帮助你。"XX怎么运行不了"，"XX卡住了"，这样的描述将不会得到回复。

在提问题前，请先阅读 [常见问题(FAQ)](https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/FAQ_en_cn)。
检查 Alas 的更新和最近的 commit，确认使用的是最新版。
上传出错 log，在 log/error 目录下，以毫秒时间戳为文件夹名，包含 log.txt 和最近的截图。若不是错误而是非预期的行为，提供在 log 目录下当天的 log和至少一张游戏截图。
